### DISCUSSION TITLE
Error in reading RDS data

### DISCUSSION THREAD

**Discussion by eeinnew** [teamalgoritma/community#105] [Q&A] [answered]
Hello, I follow the tutorial from algoritma youtube but I do not know why it is now working on me
<img width="960" alt="image" src="https://user-images.githubusercontent.com/81936292/168403293-a3818fa2-e8ac-4a91-aefb-eb4de2e52d16.png">

**Comment by inytss** [comment]
Hi @eeinnew,
If you don't mind, would you try to print just `indo_sf` object? I want to make sure whether the rds file is in the form of regular data or sf objects. Also you can try to run the code `class(indo_sf)` to make sure the type of object.

**Comment by NabiilahArdini** [comment, marked as answer]
Hi @eeinnew 

You might want to try using `str()` function on your `indo_sf` object, just to check the structure and hierarchy of the data. 
`@` is one way to access the data stored within an `S4` type data, but it said that your data is not an S4 object. 
The result of the `str()` might be useful to know which character to access the data (sometimes we use `@` sometimes we use `$` , or sometimes we need specific function from specific package for certain type of data)

I hope this answer also helps you.
